package/boot/arm-trusted-firmware-rockchip/Makefile

```diff
@@ -7,10 +7,10 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_VERSION:=2.10
+PKG_VERSION:=2.12
```

Comment:
There is 2.12.1
https://git.trustedfirmware.org/plugins/gitiles/TF-A/trusted-firmware-a.git/+/refs/tags/lts-v2.12.1

Reply:
I noticed, but it's an LTS tag and I did not see those getting used in OpenWrt before. I'll test if include/trusted-firmware-a.mk supports them. Should have done that already.

Reply:
Added a change to support A-TF LTS versions. WDYT?